<launch>
  <!-- The ROSBridge Node -->
  <include file="$(find-pkg-share rosbridge_server)/launch/rosbridge_websocket_launch.xml"/>
  <!-- The ROS web_video_server -->
  <node pkg="web_video_server" exec="web_video_server" name="web_video_server"/>

  <!-- The dummy nodes -->

  <!-- The RealSense Node -->
  <node pkg="feeding_web_app_ros2_test" exec="DummyRealSense" name="DummyRealSense"/>
  <!-- The MoveAbovePlate action -->
  <node pkg="feeding_web_app_ros2_test" exec="MoveAbovePlate" name="MoveAbovePlate"/>
</launch>